test/Telegram.Bot.Tests.Integ/Admin Bot/ChatMemberAdministrationTests.cs

```diff
@@ -125,6 +124,109 @@ public async Task Should_Receive_New_Chat_Member_Notification()
             );
         }
 
+        [OrderedFact("Should create an invite link to the group")]
+        [Trait(Constants.MethodTraitName, Constants.TelegramBotApiMethods.CreateChatInviteLink)]
+        public async Task Should_Create_Chat_Invite_Link()
+        {
+            DateTime createdAt = DateTime.UtcNow;
+
+            // Milliseconds are ignored during conversion to unix timestamp since it counts only up to
+            // seconds, so for equality to work later on assertion we need to zero out milliseconds
+            DateTime expireDate = createdAt.With(new () {Millisecond = 0}).AddHours(1);
+
+            string inviteLinkName = $"Integration tests invite link (created at {createdAt:yyyy-MM-ddTHH:mm:ss})";
```

Comment:
Fix name length: it should be 32 chars max, otherwise the test will always fail on the link name assertion